prototypes/gpt_quantum_ui/quantum_ui.py

```
@@ -0,0 +1,790 @@
+        self.wfile.write(body)
+
+    def do_POST(self) -> None:
+        # Prototype invariant:
+        # UI presentation must not execute or approve actions.
+        self.send_response(405)
+        self.send_header("Allow", "GET")
+        self.end_headers()
+
+    def log_message(
+        self,
+        format: str,
```

Ruff (0.16.1)

[error] Function argument `format` is shadowing a Python builtin

(A002)